docs/proposals/002-api-proposal/glossary.md

```diff
@@ -0,0 +1,94 @@
+# Glossary
+
+This is a glossary that attempts to more thoroughly emplain terms used within the api proposal, in an effort to give context to API decisions.
+
+<!-- toc -->
+- [API Terms](#api)
+    - [BackendPool](#backendpool)
+    - [UseCase](#UseCase)
```

Comment:
```suggestion
    - [UseCase](#usecase)
```

Reply:
updated with new naming, and dropped to lowercase